rules/numeric-host/src/lib.rs

```diff
@@ -0,0 +1,113 @@
+use anyhow::{anyhow, Ok};
+use bulwark_wasm_sdk::*;
+use lazy_static::lazy_static;
+use regex::Regex;
+
+struct NumericHostPlugin;
+
+fn check_numeric_host(req: Request) -> std::result::Result<bool, anyhow::Error> {
+    let host = req
+        .headers()
+        .get("Host")
+        .ok_or(anyhow!("Missing Host Header."))?;
+
+    lazy_static! {
+        static ref NUMERIC_HOST_REGEX: Regex =
+        Regex::new(r"^((?:(?:[0-9]+|0(x|X)[0-9a-fA-F]+|0(b|B)[0-7]+)\.){0,3}(?:[0-9]+|0(x|X)[0-9a-fA-F]+|0(b|B)[0-7]+)|\[[0-9a-fA-F:]+\])(:[0-9]{1,5})?$")
+            .unwrap();
+    }
```

Comment:
Oops, missed this in my initial review. I believe this block should move outside the function, to the top level. That way, in 0.5.0, it can compile the `Regex` only once instead of on every request.